spec/support/govuk_spec.rb

```diff
@@ -0,0 +1 @@
+GovukTest.configure
```

Comment:
I think should this be in `spec_helper`.

Reply:
Usage does say that it can be here: https://github.com/alphagov/govuk_test?tab=readme-ov-file#usage, but probably doesn't add anything. I'll move it to rails_helper (the spec_helper for this app is still the generated one).